src/LHScriptX/FeatureScriptCommands.cpp

```diff
@@ -201,6 +202,11 @@ inline glm::vec3 GetSize(int size)
 	return glm::vec3(size, size, size) * 0.001f;
 }
 
+void FeatureScriptCommands::SetATownInfluenceMultiplier(int32_t townId, float mult)
+{
+	SPDLOG_LOGGER_ERROR(spdlog::get("scripting"), "LHScriptX Function {} not implemented.", __func__);
```

Comment:
The following will kill the warnings and make it easier to debug/implement in the future.
```c+
SPDLOG_LOGGER_ERROR(spdlog::get("scripting"), "LHScriptX Function {}(townId={}, multiplier={}) not implemented.", __func__, townId, multiplier);

Not gonna block you on this, but I prefer `multiplier` to `mult`
```

Reply:
Done